events.ts

```diff
@@ -24,7 +24,7 @@ async function main() {
 
   if (contractId != null) {
     filters.push({
-      contractIds: [ contractId ]
+      contractIds: [ new SorobanClient.Contract(contractId).contractId('hex') ]
```

Comment:
This accepts either a hex, or C-strkey, and normalizes it to hex (which the RPC requires, for now). I'll open a new issue to fix that separately.